templates/common/.devcontainer/devcontainer.json/bicep-starter/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Azure Developer CLI",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": {
+            // List of images: https://github.com/devcontainers/images/tree/main/src
+            "IMAGE": "python:3.10"
+        }
+    },
+    "features": {
+        // Install Node.js and docker by default
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {
+        },
+        "ghcr.io/devcontainers/features/node:1": {
```

Comment:
why these features

Reply:
It's consistent with how the starter is modeled and taking into account of how we're trying to focus on quickstart for node and JS developers. Do you have an idea on what features should be added here?

Reply:
A perfectly fine approach is to leave out language features, and add a comment saying `// Install language features here`. I added it because python and nodeJS are generally good things to have, even if you don't work on those languages. They are pretty pervasive in so many development tooling.

Maybe remove docker?

Reply:
Eh, I'd probably err on the side of keeping this barebones. There are devs that really don't need/want node/Python.

Reply:
OK. If we want to remove features here, we would also need comment out `services` in `azure.yaml`. Otherwise, even `azd provision` fails:

![image](https://user-images.githubusercontent.com/2322434/235537032-4e8bf606-7e23-4982-9a73-1df2be76ea61.png)

I think commenting out `services` is the best thing we can do for now. Ideally, starters should have minimal comments (see `dotnet`). But `dotnet` also started out with a lot of verbose comments. What do y'all think?

Reply:
As discussed in the demo meeting, I'd prefer to comment out the services which might have language runtime dependencies so that we can keep the devcontainer config minimal and scoped to only what the user wants. The user is responsible for making sure they have the right runtimes for their services/application. It's suboptimal to include many runtimes that a user might not need (in fact, there are developers who will go out of their way to avoid certain runtimes).

Reply:
Thank you for helping drive clarity here! Will make the changes accordingly.

Reply:
Minimized features to just IaC tools.